tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -326,11 +326,16 @@ You can read more about this error on the `C# language reference <https://learn.
 Performance of C# in Godot
 --------------------------
 
-According to some preliminary `benchmarks <https://github.com/cart/godot3-bunnymark>`_,
-the performance of C# in Godot — while generally in the same order of magnitude
-— is roughly **~4×** that of GDScript in some naive cases. C++ is still
-a little faster; the specifics are going to vary according to your use case.
-GDScript is likely fast enough for most general scripting workloads.
+Comparing GDScript to C# is tricky since C# can be faster in some specific cases; generally,
+those cases are when not iterating with the engine. So using C# can bring you performance benefits,
+but it is not guaranteed.
+
+In theory, if C++ (GDNative) is used right it will always be faster than C#, but for most projects,
+the difference is negligible, so use C# or C++ if you like them and know what you are doing; otherwise,
+stick with GDScript, which is likely fast enough for most general scripting workloads.
+
+The project don't need to stick to a single language; it can use multiple of them at the same time.
+Be aware that doing so will bring its own pitfalls.
```

Comment:
```suggestion
Comparing GDScript performance to C# is tricky, since C# can be faster in some specific cases. However, C#'s performance can also be brought down by garbage collection which occurs at random and unpredictable moments. This can result in difficult-to-diagnose stuttering issues in complex projects, and is not exclusive to Godot. As a result, C# may result in a performance benefit for some number crunching scenarios, but it is not guaranteed.

For most projects that don't need to perform intensive computations, the difference is negligible. Use C# if you like it and are familiar with the language; otherwise,
stick with GDScript, which is likely fast enough for most general scripting workloads.
Also, remember that using a faster scripting language will not help improve performance
in GPU-bottlenecked scenarios, nor in scenarios where the bottleneck is within the engine (such as physics or navigation).

Your project doesn't need to stick to a single language. It can use multiple of them at the same time.
Be aware that doing so will bring its own pitfalls, as covered in :ref:`doc_cross_language_scripting`.
```

Reply:
I don't see how C++ is out of scope, the topic is "Performance of C# in Godot", GDNative is integrated in the engine so...

Also I fear that don't adding C++ in the equation can make people come asking "Is C++ faster than C#? The docs don't say anything about it".

I agree that C++ comes out of nowhere, but I couldn't find a better way of writing "C++ _can_ be faster than C#, but I doubt you would need that".